plugin/src/main/java/org/jenkinsci/plugins/structs/describable/DescribableModel.java

```diff
@@ -267,12 +271,20 @@ public String getDisplayName() {
      * or {@link Class#getSimpleName} may be used in case the argument type is {@link Describable}
      * and only one subtype is registered (as a {@link Descriptor}) with that simple name.
      */
-    public T instantiate(Map<String,?> arguments) throws Exception {
+    public T instantiate(Map<String,?> arguments) throws IllegalArgumentException {
+        return instantiate(arguments, new LogTaskListener(LOGGER, Level.FINE));
```

Comment:
```suggestion
        return instantiate(arguments, new LogTaskListener(LOGGER, Level.WARNING));
```

under the assumption that any cases of a message being printed are, in fact, warnings.